betalectic/betalectic.github.io
# Added tailwindcss to the project

### **PR Template:**

---

### **Title:**
`Add Tailwind CSS to the project`

---

### **Description:**
This PR introduces Tailwind CSS to the project to enable utility-first CSS styling and simplify the creation of custom, responsive, and consistent design across the project.

#### Key changes:
- Installed and configured **Tailwind CSS** with PostCSS and Autoprefixer.
- Integrated Tailwind into the project’s PostCSS pipeline.
- Added a basic configuration for customizing Tailwind settings in the future (`tailwind.config.js`).
- Updated existing CSS where necessary to use Tailwind utility classes for testing purposes.
- Verified that Tailwind classes work in `.md` files, enabling usage in Markdown files.

---

### **Type of Change:**
- [x] Feature (a new functionality)
- [ ] Chore (maintenance or a refactor)

---


### **How to Test:**
1. Pull the branch: `git checkout feature/add-tailwindcss`
2. Install dependencies: `yarn install`
3. Start the development server: `yarn start`
4. Verify Tailwind CSS is working by inspecting the usage of utility classes and confirming the expected design.
5. Test Tailwind utility classes in markdown files (such as in the `docs` or `blog` pages).



This template will help guide reviewers and team members through the changes you’ve made, ensuring a smooth PR process. You can modify it as needed based on your project or team requirements.
